main.go

```diff
@@ -48,6 +49,12 @@ func main() {
 		case "version":
 			println("LazySQL version: ", version)
 			os.Exit(0)
+		default:
+			err := components.InitFromArg(argsWithProg[1])
```

Comment:
I'm doubtful about how the code react if you pass more than one argument, the vode doesn't seem logic now

Reply:
I'm not sure what should be done in this scenario, the user will connect to the first database passed there and ignore the rest. Should I just then make a check if number of arguments is exactly two and display an error if it's too many?

Reply:
I think you should report an error if there are more than 2 arguments.

Also use could do strange things such `ladygit version somealias`

Reply:
I would argue that the version "subcommand" could be turned into a -v or -version flag. I don't think there is a standardized way that golang clis follow when it comes to things like these. For [lazygit](https://github.com/jesseduffield/lazygit), version is displayed via the -v or --version command, [lf](https://github.com/gokcehan/lf/tree/master) has it as -version flag. I'd say most of the cli's in general have it as the -v flag and maybe we should make the change for lazysql as well. Extra factor to take into account is that now with the patch is that if someone were to have a sqlite database called _version_ and wanted to connect to it directly, it wouldn't work as expected (realistically not the gravest issue but hey).

Reply:
I agree with you.

But no matter how the version is handled, the code should handle multiple argument as an error

Reply:
I just mentioned it as it changes the way we parse positional arguments. Made the appropriate changes so that it errors out on multiple connection urls.